.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.0
```

Comment:
@bennacer860 Thanks for the PR. Can you remove the `.ruby-version` from this commit and it should be good to go. I don't mind  the adding of byebug, but I don't think that the .ruby-version is as important here since this gem does work with older versions of Ruby. Thanks!